homeassistant/components/iron_os/entity.py

```diff
@@ -9,30 +9,31 @@
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
 from .const import MANUFACTURER, MODEL
-from .coordinator import IronOSBaseCoordinator
+from .coordinator import IronOSLiveDataCoordinator
 
 
-class IronOSBaseEntity(CoordinatorEntity[IronOSBaseCoordinator]):
+class IronOSBaseEntity(CoordinatorEntity[IronOSLiveDataCoordinator]):
     """Base IronOS entity."""
 
     _attr_has_entity_name = True
 
     def __init__(
         self,
-        coordinator: IronOSBaseCoordinator,
+        coordinator: IronOSLiveDataCoordinator,
         entity_description: EntityDescription,
         context: Any | None = None,
```

Comment:
What is the point of this argument?

Reply:
Hmm, I think that one is also not needed anymore, the context is passed to the settings coordinator in a different way. I will check that later